lib/std/unicode.zig

```diff
@@ -32,7 +82,7 @@ pub fn utf8ByteSequenceLength(first_byte: u8) !u3 {
 /// out: the out buffer to write to. Must have a len >= utf8CodepointSequenceLength(c).
 /// Errors: if c cannot be encoded in UTF-8.
 /// Returns: the number of bytes written to out.
-pub fn utf8Encode(c: u21, out: []u8) !u3 {
+pub fn utf8Encode(c: u32, out: []u8) !u3 {
```

Comment:
Why change to a `u32` here?

Reply:
Because it is a function. I will look into the LLVM optimization bug.